phpstan.neon

```diff
@@ -26,6 +26,10 @@ parameters:
         - %currentWorkingDirectory%/*/Tests/app/var/*
         # RestController extends FOSRestController which was removed in friendsofsymfony/rest-bundle:3.0
         - %currentWorkingDirectory%/src/Sulu/Component/Rest/RestController.php
+        # SwiftMailer was removed and con not be analyzed by phpstan anymore
+        - %currentWorkingDirectory%/src/Sulu/Bundle/SecurityBundle/Tests/Functional/Controller/LegacyResettingControllerTest.php
+        - %currentWorkingDirectory%/src/Sulu/Bundle/MarkupBundle/Listener/SwiftMailerListener.php
+        - %currentWorkingDirectory%/src/Sulu/Bundle/MarkupBundle/Tests/Unit/Listener/SwiftMailerListenerTest.php
```

Comment:
PHPStan can not analyze this and not add errors to the baseline here:

<img width="839" alt="Bildschirmfoto 2022-04-22 um 13 10 06" src="https://user-images.githubusercontent.com/1698337/164703573-03b39b8d-c4a9-4942-910c-88cb20a5eec8.png">

Reply:
👍